compiler/rustc_codegen_ssa/src/mir/operand.rs

```diff
@@ -131,7 +131,7 @@ impl<V: CodegenObject> OperandValue<V> {
 /// to avoid nasty edge cases. In particular, using `Builder::store`
 /// directly is sure to cause problems -- use `OperandRef::store`
 /// instead.
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, Debug)]
```

Comment:
literally already has one. if you need the Debug to be different then weaken the `V: CodegenObject` bound.

```suggestion
#[derive(Copy, Clone)]
```

Reply:
My bad!